.travis.yml

```diff
@@ -1,11 +1,30 @@
 os: osx
 # No version of Python is available via virtualenv on OS X workers, see https://github.com/travis-ci/travis-ci/issues/2312
 language: generic
+sudo: required
 
 env:
   - TOX_ENV=py27-first_startup
   - TOX_ENV=py35-first_startup
 
+before_install:
+  # Install Singularity
+  - sudo apt install -y build-essential libssl-dev uuid-dev libgpgme11-dev squashfs-tools wget git
```

Comment:
We are not using travis for these tests anymore. The docker testing image comes pre-installed with singularity. For a simple test I'd suggest you create something unit-test like in `test/integration`, which will be picked up automatically.